code/index.html

```diff
@@ -4,27 +4,54 @@
     <meta charset="utf-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Project 3</title>
-    <!-- dont forget to add a css file and link it here! -->
+    <link rel="stylesheet" href="style.css"> 
   </head>
   <body>
-    <!-- video or image as a header is cool :) -->
 
-    <!-- Signup form -->
+   <header class="header">
+     <h1>Bake a cake</h1>
+    
+    </header> 
+     
+   <main>
+    
+  <section class="input-container">
+   
+      <form id="form" action="http://httbin.org/anything" method="post">
+     <label> First name </label>
+     <input id="" class="form" type="text" name="name" required>
+     
+     <label>Last name</label>
+     <input id="" class="form" type="text" name="last name" required>
+     
+     <label>E-mail</label>
+     <input class="form" type="email" name="E-mail"required>
+  
+    <button class="sendapplication" type="submit">Sign up for our baking course!</button>
```

Comment:
Perhaps this class could be renamed "send-application" just to make it easier to read?